automation.py

```diff
@@ -22,13 +22,14 @@
 python_path = sys.executable
 script_path = os.path.realpath(__file__)
```

Comment:
can we delete line 22 and 23 we dont need them anymore

Reply:
Just pushed those changes up.